core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_body_directive.html

```diff
@@ -1,13 +1,13 @@
 <script type="text/ng-template" id="simpleEditor/body">
   <div class="main">
     <div class="content">
-      <h3>What would you like to teach?</h3>
+      <h3 id="title" scroll-sync>What would you like to teach?</h3>
```

Comment:
Might not need the IDs here (see comments above about the highlight). Could be cleaner to do everything exclusively through scroll-sync; what do you think?